carla_integration/docker-compose.yml

```diff
@@ -48,6 +48,39 @@ services:
       - /opt/carma/yolo:/opt/carma/yolo
     command: bash -c 'wait-for-it.sh localhost:11311 -- roslaunch /opt/carma/vehicle/config/carma_docker.launch'
 
+  platform_ros2:
+    image: usdotfhwastoldev/carma-platform:foxy-develop
+    network_mode: host
+    container_name: platform_ros2
+    volumes_from:
+      - container:carma-config:ro
+    volumes:
+      - /opt/carma/logs:/opt/carma/logs
+      - /opt/carma/.ros:/home/carma/.ros
+      - /opt/carma/vehicle/calibration:/opt/carma/vehicle/calibration
+      - /opt/carma/maps:/opt/carma/maps
+      - /opt/carma/routes:/opt/carma/routes
+      - /opt/carma/yolo:/opt/carma/yolo
+    command: bash -c 'source /opt/carma/install_ros2/setup.bash && ros2 launch /opt/carma/vehicle/config/carma_docker.launch.py'
+
+  ros1_bridge:
+    image: usdotfhwastoldev/carma-msgs:foxy-develop
+    network_mode: host
+    container_name: ros1_bridge
```

Comment:
I'm not sure I recall what our final decision was RE: docker architecture for this ROS2 stuff, is there a document that spells out how we're separating the two, and what's running in each environment?

Reply:
Not a description on the per node level, but in the ROS2 Migration Plan and Architecture on confluence we state that the system will be migrated on a subsystem basis. So first object detection, then v2x, then localization etc. The exact boundary comes down to what the system looks like when we are moving a given subsystem.